toolkit/scripts/toolchain.mk

```diff
@@ -195,7 +196,8 @@ $(toolchain_rpms): $(TOOLCHAIN_MANIFEST) $(final_toolchain) $(STATUS_FLAGS_DIR)/
 		mkdir -p $(dir $@) && \
 		tar -I $(ARCHIVE_TOOL) -xvf $(final_toolchain) -C $(dir $@) --strip-components 1 built_rpms_all/$(notdir $@) && \
 		touch $@ ; \
-	fi || $(call print_error, $@ failed) ;
+	fi || $(call print_error, $@ failed) ; \
+	touch $@
```

Comment:
I'm curious, if this changes also makes `REFRESH_WORKER_CHROOT` unnecessary - it was a quick workaround to prevent the annoying rebuilds of the chroot when none of the toolchain packages got modified. Basically, if the chroot doesn't get rebuilt now every single time we build a package and is only built when it either doesn't exist or one of its RPMs changed, I think we can remove it to reduce the clutter. I've introduced it and would be happy to see it go away.;)

Reply:
I'd say leave it in for now. I can't say 100% that this will fix all rebuilds, just the ones I was able to repro during my work. We can revisit in a few weeks after seeing if it actually fixed it.